helpers.cpp

```diff
@@ -19,8 +19,76 @@ void reflect(int height, int width, RGBTRIPLE **image) {
 // Definition for Blur Filter
 void blur(int height, int width, RGBTRIPLE **image) {
     // code to be written
+    
+    int sumBlue;
```

Comment:
```suggestion
    int sumBlue {0};
```
Do this for rest variables